core/src/main/java/org/pentaho/di/osgi/OSGIPluginTracker.java

```diff
@@ -199,6 +199,13 @@ public void setProxyUnwrapper( ProxyUnwrapper proxyUnwrapper ) {
         this.proxyUnwrapper = proxyUnwrapper;
     }
 
+    public ProxyUnwrapper getProxyUnwrapper() {
```

Comment:
So looks like there has to be a ProxyUnwrapper registered in the system for the bridge to work.It would be nice to fall-back to a NoOpProxyUnwrapper if none is available which simply assumes no proxy and returns the service object.